.github/workflows/ci-image-diff-sync.yml

```diff
@@ -50,10 +50,10 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
       with:
-        python-version: 3.9.9
+        python-version: '3.9.9'
```

Comment:
Even though we have node version specified we are still getting the warning. I'm guessing it's because we are missing quotes? 

https://github.com/actions/setup-python#basic-usage